docs/tutorials/iers_a.rst

```diff
@@ -0,0 +1,70 @@
+:orphan:
+
+.. doctest-skip-all
+
+**********************************
+Using and updating IERS Bulletin A
+**********************************
+
+Contents
+========
+
+* :ref:`iers_a-Background`
+* :ref:`iers_a-What_are_IERS_Bulletins_A_and_B`
+* :ref:`iers_a-How_do_I_get_IERS_Bulletin_A_for_astroplan?`
+
+.. _iers_a-Background:
+
+Background
+==========
+
+Keeping accurate, consistent time systems is rough. We intuitively want
+time to be described by a continuous cyclic clock like the clock on your
+wall or the calendar, but the Earth's rate of rotation varies measurably
+on human timescales. As a result, the time measured by an atomic clock and
+the time that you would infer from measuring how long it has been since the
+last solar noon are constantly becoming offset from one another with
+stochastic changes in Earth's moment of inertia and slowly acting tidal forces.
+
+For this reason, there is a time system that keeps track of seconds like an
+atomic clock, UTC, one that matches up with the Earth's rotation, UT1. The
+difference between UT1 and UTC is constantly changing as the Earth's rotation
+changes and as leap seconds get added to UTC to compensate.
+
+In order to accurately predict the apparent position of a celestial object from
+the Earth, for example in altitude and azimuth coordinates, one needs to know
+the orientation of the Earth at any point in time, and since the Earth's
+rotation does not change in a predictable way, we must rely on observations
+of the Earth's orientation as a function of time to accurately calculate
+UT1-UTC.
+
+.. _iers_a-What_are_IERS_Bulletins_A_and_B:
+
+What are IERS Bulletins A and B?
+================================
+
+The International Earth Rotation and Reference Systems Service (IERS) is
+responsible for measuring the Earth's orientation as a function of time, and
+making predictions for the Earth's orientation in the near future (accounting
+for scheduled leap seconds). The data products released by the IERS used by
+astroplan are the IERS Bulletins A and B. IERS Bulletin B is a table with Earth
```

Comment:
The title of this section is "What are A and B", but then you only say what "B" is.
My suggestion would be to make two bullets here, one for A and B, saying what they are and how they differ.
Maybe also link to http://astropy.readthedocs.org/en/latest/time/index.html?highlight=iers#transformation-offsets ?